index.js

```diff
@@ -65,7 +55,7 @@ Converter.prototype.toBase64 = function () {
 
 Converter.prototype.toComment = function (options) {
   var base64 = this.toBase64();
-  var data = 'sourceMappingURL=data:application/json;base64,' + base64;
+  var data = 'sourceMappingURL=data:application/json;charset=utf-8;base64,' + base64;
```

Comment:
This is from Generator.prototype.inlineMappingUrl of your project `inline-source-map` that adds charset even if it not specified